config/generate-keypair/src/lib.rs

```diff
@@ -17,7 +18,10 @@ pub fn create_faucet_key_file(output_file: &str) -> KeyPair<Ed25519PrivateKey, E
         panic!("Specified output file path is a directory");
     }
 
-    let (private_key, _) = compat::generate_keypair(None);
+    let mut seed_rng = OsRng::new().expect("can't access OsRng");
+    let mut rng = rand::rngs::StdRng::from_seed(seed_rng.gen());
+
+    let (private_key, _) = compat::generate_keypair(&mut rng);
```

Comment:
you should be able to pass `seed_rng` directly here - no need to use another rng